src/util/belt.rs

```diff
@@ -105,29 +110,23 @@ impl StagingBelt {
             .get_mapped_range_mut()
     }
 
-    /// Produce a command buffer with all the accumulated transfers.
+    /// Prepare for command buffers to be submitted.
```

Comment:
since there is nothing about "command buffers" in the signature, we shouldn't be documenting it like that.
Maybe "prepare mapped buffers to be used in submission" or something like that?